app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -649,10 +511,220 @@ def test_serializer_rejects_too_many_primary_probe_sets(self):
             serializer.errors["primary_probe_set_ids"][0],
         )
 
+    def test_does_not_delete_probesets_when_other_fields_specified(self):
```

Comment:
Always get a little nervous when a refactor also comes with big test diffs, but as far as I can tell they're just additions and moves without big changes. 🤷‍♂️

Reply:
Yeah I was trying to pad out the tests to explicitly validate that we're only updating the fields we care about in this one serializer to try to catch cases like the one we saw last week where updating unrelated fields was clearing out feature_config, for example.  Because there was actually a lot of similar cases where updating one page would clear out values in another, so I wanted to make sure the tests were as exhaustive for those cases as I could.